src/pool_allocator.c

```diff
@@ -50,8 +50,8 @@ MEMKIND_EXPORT void *pool_allocator_malloc(PoolAllocator *pool, size_t size)
 
 MEMKIND_EXPORT void *pool_allocator_malloc_pages(PoolAllocator *pool,
                                                  size_t size,
-                                                 uintptr_t *address,
-                                                 size_t *nof_pages)
+                                                 uintptr_t address[2],
```

Comment:
Arrays of sizes equal to 3 are passed here in a call found in mtt_internals_malloc().